integration/local-gadget/Makefile

```diff
@@ -0,0 +1,81 @@
+MINIKUBE_VERSION ?= v1.27.0
+KUBERNETES_VERSION ?= v1.24.6
+MINIKUBE_DRIVER ?= docker
+
+CONTAINER_RUNTIME ?= docker
+
+# make does not allow implicit rules (with '%') to be phony so let's use
+# the 'phony_explicit' dependency to make implicit rules inherit the phony
+# attribute
+.PHONY: phony_explicit
+phony_explicit:
+
+# minikube
+
+MINIKUBE = $(shell pwd)/bin/minikube-$(MINIKUBE_VERSION)
+.PHONY: minikube
+minikube:
+	mkdir -p bin
+	test -e bin/minikube-$(MINIKUBE_VERSION) || \
+	(cd bin && curl -Lo ./minikube-$(MINIKUBE_VERSION) https://github.com/kubernetes/minikube/releases/download/$(MINIKUBE_VERSION)/minikube-linux-amd64)
+	chmod +x bin/minikube-$(MINIKUBE_VERSION)
+
+# build
+
+build:
+	make -C $(shell pwd)/../.. local-gadget
+
+# clean
+
+.PHONY: clean
+clean:
+	$(MINIKUBE) delete -p minikube-docker
+	$(MINIKUBE) delete -p minikube-containerd
+	$(MINIKUBE) delete -p minikube-cri-o
+	rm -rf bin
+
+# setup
+
+SETUP_TARGETS = \
+	setup-docker \
+	setup-containerd \
+	setup-cri-o
+
+.PHONY: setup-all
+setup-all: $(SETUP_TARGETS) setup
+
+setup: setup-$(CONTAINER_RUNTIME)
+
+.PHONY: phony_explicit
+setup-%: minikube
+	docker inspect minikube-$* >/dev/null 2>&1 || \
+	$(MINIKUBE) start -p minikube-$* --driver=$(MINIKUBE_DRIVER) --kubernetes-version=$(KUBERNETES_VERSION) --container-runtime=$* --wait=all
+
+# test
+
+TEST_TARGETS = \
+	test-docker \
+	test-containerd \
+	test-cri-o
+
+.PHONY: test-all
+test-all: $(TEST_TARGETS) test
+
+test: test-$(CONTAINER_RUNTIME)
+
+.PHONY: phony_explicit
+test-%: build
```

Comment:
Could you please add here a comment with an example of how `INTEGRATION_TESTS_PARAMS` can be used like you did for the inpektor-gadget tests?

```bash
# INTEGRATION_TESTS_PARAMS="-test.run TestListContainers" CONTAINER_RUNTIME=containerd make -C integration/local-gadget test
```